### ISSUE TITLE
Postgres jdbc-driver issue when using two conditionals inside anonymous code block

### ISSUE THREAD

**Issue by MarcoBorchert** [serge-rider/dbeaver#2206]
Using two (or more) conditionals inside an anonymous code block  produces "Internal jdbc driver error java.lang.ArrayIndexOutOfBoundsException" when executed via 'Execute SQL Script" (in 4.2.1; works in 4.2.0).

```
DO LANGUAGE plpgsql 
$$	
BEGIN
	IF FALSE THEN
	END IF;
	IF FALSE THEN	
	END IF;
END$$
```

Workaround: select whole query and execute it (with ctrl+enter).

**Comment by serge-rider** [comment]
Seems to be duplicate of #2199.
Try to update PG driver to version 42.x.

**Comment by MarcoBorchert** [comment]
Using PG driver 42.1.4 the error message changes: "Unterminated dollar quote started at position 21 [...]".
Workaround executing with ctrl+enter still works.

**Comment by serge-rider** [comment]
That was fixed in #2171
